opencti-platform/opencti-front/src/private/components/entities/systems/SystemKnowledge.js

```diff
@@ -78,6 +78,28 @@ class SystemKnowledgeComponent extends Component {
             ))
             }
           />
+          <Route
+            exact
+            path="/dashboard/entities/systems/:systemId/knowledge/targets"
+            render={(routeProps) => (
+              <EntityStixCoreRelationships
+                entityId={system.id}
+                relationshipTypes={['targets']}
+                isRelationReversed
+                entityLink={link}
+                allDirections={true}
+                stixCoreObjectTypes={[
```

Comment:
According to the stix.ts file, the tool entity is not available.

Reply:
Fixed

Reply:
The list still not aligned with the stix.ts file. Maybe some relationship is missing in this file.

Existing
ENTITY_TYPE_CAMPAIGN
ENTITY_TYPE_INCIDENT
ENTITY_TYPE_INTRUSION_SET
ENTITY_TYPE_MALWARE
ENTITY_TYPE_THREAT_ACTOR

Your adding
ENTITY_TYPE_TOOL

Reply:
FIxed